### PR TITLE
feat: implement README generator for OSS projects

### PR DESCRIPTION
**Pull request by cprice404**
This commit modifies the template github action to have a real
implementation.  It now reads in README.md.template, asserts
that your template includes a variable to insert the OSS header
at the top, and then generates the README.md file with a consistent
header based on the template.

### CODE REVIEW

File: github-actions/oss-readme-template/__tests__/main.test.ts [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
-});
+// test('test runs', () => {
+//   process.env['INPUT_MILLISECONDS'] = '500';
+//   const np = process.execPath;
+//   const ip = path.join(__dirname, '..', 'lib', 'main.js');
+//   const options: cp.ExecFileSyncOptions = {
+//     env: process.env,
+//   };
+//   console.log(cp.execFileSync(np, [ip], options).toString());
+// });
```
Comment (eaddingtonwhite) [inline github-actions/oss-readme-template/__tests__/main.test.ts:20]: mean to leave commented? Maybe just remove dummy test since you added your own?

**Review by eaddingtonwhite — DISMISSED** [review]
Nice work this looks great with unit test as well 💯 one nit on commented out test but looks good to me. Excited see this rolled out.

File: github-actions/oss-readme-template/src/readme.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+interface HeaderTemplateContext {
+  projectStatus: string;
+  projectStability: string;
+}
+
+const OSS_README_HEADER_TEMPLATE = `<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-{{ projectStatus }}.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-{{ projectStability }}.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
```
Comment (eaddingtonwhite) [inline github-actions/oss-readme-template/src/readme.ts:41]: nit maybe move `momentohq` gh org to constant in case changes down line

**Review by kvcache — APPROVED** [review]
This is neat. I don't see anything wrong but it's also quite a lot of surface area 😄

File: github-actions/oss-readme-template/__tests__/main.test.ts (hunk 1 above)
Comment (cprice404) [reply github-actions/oss-readme-template/__tests__/main.test.ts:20]: yeah I meant to leave that in, I would never be able to figure out how to write that test without an example if we ever do want to resuscitate it.  I put a comment above explaining why I was leaving it in.

**Review by eaddingtonwhite — APPROVED** [review]

**Pull request merged**